.github/workflows/workflow.yml

```diff
@@ -53,12 +53,19 @@ jobs:
     - uses: actions/checkout@v2
       with:
           submodules: 'recursive'
+    - name: Install SonarCloud's dependencies for C++ analysis
+      run : |
+        apt update
+        apt install -y unzip wget
+        wget 'https://sonarcloud.io/static/cpp/build-wrapper-linux-x86.zip' 'https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.6.0.2311-linux.zip'
+        unzip -u build-wrapper-linux-x86.zip
+        unzip -u sonar-scanner-cli-4.6.0.2311-linux.zip
     - name: Install dependencies for python2
       run: pip install -r source/jormungandr/requirements_dev.txt
     - name: configure
       run: cmake -DSTRIP_SYMBOLS=ON source
-    - name: make
-      run:  make -j $(nproc)
+    - name:
+      run: build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir build-wrapper-output-dir make -j $(nproc)
```

Comment:
The build-wrapper is a tool used by sonaqube to discover all files being compiled with there associated configuration (eg. #define etc... ). That's why it wraps the call to `make`